### ISSUE TITLE
AVT -- High contrast test for Dropdown

### ISSUE THREAD

**Issue by lilyclyc** [carbon-design-system/carbon-components-react#117]
![image](https://user-images.githubusercontent.com/30255650/28509898-8e39b692-7077-11e7-82e8-4e238dd9ecac.png)

**Comment by hellobrian** [comment]
Is this the IBM Visualization tool you're using? FWIW, we're relying on aXe for a11y checks now.

The reason why this is failing is because the a11y tool you're using is not accounting for the resulting background colors. In other words, it's being thrown off by the `background-color` is using a transparent `rgba` color `rgba(61, 112, 178, 0.1)`, which actually resolves to `#DEE5F2` or `rgb(222, 229, 242)`. All of our background colors have been tested to meet AA contrast ratios.

So, when we pair the resulting background color and text color, we get a contrast ratio of 11.9.

![image](https://user-images.githubusercontent.com/4185382/28532258-ee660d5e-705e-11e7-87eb-920115d87f38.png)

Source: [leaverou.github.io/contrast-ratio](http://leaverou.github.io/contrast-ratio/#%23152935-on-%23DEE5F2)

**Comment by lilyclyc** [comment]
@hellobrian -- Could you give the url link for aXe tool?

**Comment by hellobrian** [comment]
[aXe chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
